beaker.gemspec

```diff
@@ -48,6 +48,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'stringify-hash', '~> 0.0'
   s.add_runtime_dependency 'beaker-hiera', '~> 0.0'
   s.add_runtime_dependency 'beaker-hostgenerator'
+  s.add_runtime_dependency 'beaker-puppet', '~> 0.0'
```

Comment:
won't this only install 0.x versions, requiring a change later when these gems get to 1.x+?

sometimes i use `~> 0`